java/bench/src/java/org/apache/orc/bench/Driver.java

```diff
@@ -23,7 +23,6 @@
 import org.apache.commons.cli.HelpFormatter;
 import org.apache.commons.cli.Options;
 import org.apache.commons.cli.ParseException;
-import org.apache.hadoop.conf.Configuration;
```

Comment:
In general, you shouldn't fix imports in files that your patch isn't touching. Go ahead and leave them in this patch.